views/layout/footer.ejs

```diff
@@ -1,3 +1,9 @@
 <footer>
-  <!-- ADD SM ICONS -->
+  <!-- Icon source: https://www.iconfinder.com/neilorangepeel -->
+  <img src="/img/iconfinder_facebook_square_black_107119.png" alt="facebook icon">
+  <img src="/img/iconfinder_instagram_square_black_107097.png" alt="instagram icon">
+  <!-- Copyright HTML symbol: https://www.w3schools.com/html/html_symbols.asp -->
+  <p>&#169; Team Charmander 2020</p>
```

Comment:
Great! Make a follow up Asana Task for adding an a tag around this that links to our GitHub Team Repo

Reply:
Of course! Thank you for catching that!